pkg/vpcmodel/semanticDiffSubnets.go

```diff
@@ -203,20 +200,39 @@ func (subnetDiff *SubnetsDiff) EnhancedString(thisMinusOther bool) string {
 	return res
 }
 
-func diffDescription(diff DiffType) string {
+// prints connection for func (subnetDiff *SubnetsDiff) EnhancedString(..) where the connection could be empty
+func connStr(conn *common.ConnectionSet) string {
+	if conn == nil {
+		return "No connection"
+	}
+	return conn.EnhancedString()
+}
+
+func diffAndWorkLoadDisc(diff DiffType, src, dst VPCResourceIntf, thisMinusOther bool) (diffDisc, workLoad string) {
+	const (
+		WorkloadDiffInfo = ", workloads-diff-info: workload"
```

Comment:
There is no term `workload` here.. for the subnets analysis it is `subnet`, and for endpoints it is `network interface`

Reply:
I asked you whether this is the term to use, and I thought you said yes. Clearly this was a misunderstanding - I'll fix. It certainly makes more sense this way.